src/@types/Asset.ts

```diff
@@ -89,6 +89,20 @@ export interface AssetDatatoken {
   serviceId: string
 }
 
+export interface Stats {
+  /**
+   * How often an asset was consumed, meaning how often it was either downloaded or used as part of a compute job.
+   * @type {number}
+   */
+  orders: number
+
+  /**
+   * Total amount of veOCEAN allocated on this asset.
+   * @type {number}
+   */
+  allocated?: number
```

Comment:
why is it optional? Is it supposed to be on all assets, or not? Cause if it does't have allocations, it should have `0` as value